### DISCUSSION TITLE
Translation?

### DISCUSSION THREAD

**Discussion by xupisco** [iaincollins/icarus#18] [Ideas & Suggestions]
Hi there, is it possible to add/create a translation for Icarus?
I may help with pt-br.

If so, let me know how to do it. :)

**Comment by iaincollins** [comment]
Thanks for offering!

It doesn't currently have support for translations. This was a tough decision because it's generally regarded as easier to add them when designing an app with them in mind than it is to retro-fit them later in development.

The challenge I had was I didn't know if I'd be able to make this into an interesting app at all when I started and it would have been a lot of extra work / scaffolding I didn't have time to do [ in addition to actual feature development ]. Also things were changing really rapidly so translations would have been made redundant all the time.

While I've only done a small number of the things I'd like to in the app so far, I think my position on that has changed and have been thinking recently how nice to have translations, especially as Elite itself is available in multiple languages.

When I get back from holiday in a couple of weeks I'll do an experiment to see what that might look like and let you know! I'll reply here and tag you. :-)

**Reply by HohMac** [reply]
That would be really cool.
I can then offer the translation for-de-de
